src/clients/c++/request.h

```diff
@@ -329,6 +329,23 @@ class InferContext {
     /// \param input The vector holding tensor string values.
     /// \return Error object indicating success or failure.
     virtual Error SetFromString(const std::vector<std::string>& input) = 0;
+
+    /// Set tensor values for this input by reference into a shared memory
+    /// region. The values are not copied and so it must not be modified or
+    /// destroyed until this input is no longer needed (that is until the Run()
+    //// call(s) that use the input have completed). For batched inputs this
+    /// function must be called batch-size times to provide all tensor values
+    /// for a batch of this input.
+    /// \param shm_key The identifier for the shared memory object that is used
```

Comment:
With Register function this changes to the user-supplied name for the register shared memory region.

Reply:
Yes